docs/documentation/sitespeed.io/configuration/index.md

```diff
@@ -147,10 +147,13 @@ s3
   --s3.key                The S3 key
   --s3.secret             The S3 secret
   --s3.bucketname         Name of the S3 bucket
-  --s3.path               Override the default folder path in the bucket where the results are uploaded. By default it's "DOMAIN_OR_FILENAME/TIMESTAMP", or the name of the
-                          folder if --outputFolder is specified.
+  --s3.path               Override the default folder path in the bucket where the results are uploaded. By default it's "DOMAIN_OR_FILENAME/TIMESTAMP", or the name of the folder if --outputFolder is specified.
   --s3.region             The S3 region. Optional depending on your settings.
-  --s3.removeLocalResult  Remove all the local result files after they have been uploaded to S3                                                     [boolean] [default: false]
+  --s3.endpoint           S3 endpoint. Defaults to s3.amazonaws.com
+  --s3.ssl                Enable or not SSL for S3                                                                                                                                                                    [boolean] [default: true]
```

Comment:
If the S3 service is hosted internally, some do not have SSL or have a self signed certificate.